modules/@apostrophecms/schema/lib/addFieldTypes.js

```diff
@@ -1051,12 +1051,6 @@ module.exports = (self) => {
         // Handy in CSV: allows titles or _ids
         input = input.split(/\s*,\s*/);
       }
-      if (field.min && field.min > input.length) {
```

Comment:
I moved this logic to the end so I'm testing the number of actual related documents found and not just the untrusted input.